src/main/java/net/marcellperger/mathexpr/parser/Parser.java

```diff
@@ -69,9 +67,10 @@ public MathSymbol parse() throws ExprParseException {
         return parseDoubleLiteral_null();
     }
 
+    private static final int PREC_IN_PARENS = 3;  // TODO compute max prec level
     public @Nullable MathSymbol parseParens() throws ExprParseException {
         advanceExpectNext('(');
-        MathSymbol sym = parseInfixPrecedenceLevel(2);  // TODO use .parse() when it is completed
+        MathSymbol sym = parseInfixPrecedenceLevel(PREC_IN_PARENS);  // TODO use .parse() when it is completed
```

Comment:
This `TODO` is wrong - `.parse()` checks for that is reached EOF etc. which we don't want. Either add a `_parse()`/similar or get rid of this TODO.

Reply:
Or this may be better in another PR?